stripe/src/main/java/com/stripe/android/paymentsheet/ui/BaseSheetActivity.kt

```diff
@@ -107,17 +107,14 @@ internal abstract class BaseSheetActivity<ResultType> : AppCompatActivity() {
     }
 
     private fun updateToolbarButton(isStackEmpty: Boolean) {
-        if (isStackEmpty) {
-            toolbar.navigationIcon =
-                ContextCompat.getDrawable(this, R.drawable.stripe_paymentsheet_toolbar_close)
-            toolbar.navigationContentDescription =
-                resources.getString(R.string.stripe_paymentsheet_close)
+        val assets = if (isStackEmpty) {
+            Pair(R.drawable.stripe_paymentsheet_toolbar_close, R.string.stripe_paymentsheet_close)
```

Comment:
I'm not a fan of using `Pair` in this way because the meaning of `first` and `second` quickly become lost. Can we create a small data class instead?

Reply:
Done